src/components/Table/styled.tsx

```diff
@@ -10,24 +11,46 @@ const ROW_HEIGHT = 56;
 
 export const Container = styled.div`
   ${boxSizing};
-  display: block;
+  display: flex;
+  flex-direction: column;
   max-width: 100%;
+  overflow: hidden;
 `;
 
 export const Scroll = styled.div`
-  display: block;
+  display: flex;
+  flex-grow: 1;
   max-width: 100%;
-  overflow-x: auto;
-  overflow-y: hidden;
+  overflow: auto;
   scroll-behavior: smooth;
 `;
 
 export const Table = styled.table`
   width: 100%;
   border-spacing: 0;
+  z-index: ${({ theme }) => theme.honeycomb.zIndexes.normal};
 `;
 
-export const Thead = styled.thead``;
+interface HeaderProps {
+  fixed: boolean;
+  background?: Property.Color;
```

Comment:
I feel that we can just use backgroundColor here? cause usually we can also set things like image URL for `background`. Or maybe Property.Background?

Reply:
Good point, changed!